vimiv/imutils/exif.py

```diff
@@ -99,40 +100,42 @@ def _load_exif(self, filename):
         """Load exif information from filename into the dictionary."""
         try:
             self._exif = piexif.load(filename)
+            desired_keys = [
+                e.lower().strip()
+                for e in api.settings.get_value("metadata.keylist").split(",")
```

Comment:
If the setting was stored in a variable, you could refer to it here via the object. I.e.:
```
for e in api.settings.metadata.keylist.value.split(", ")
```
I admit this is quite a list of `.` chained stuff, but at least we don't rely on the string.

Reply:
I see, where it comes handy to store the setting in a variable. :blush:  However, I would suggest to add an`.strip()`.This way it is more robust when do user does not get the formatting fully right